cmd/otelcontribcol/go.mod

```diff
@@ -4,7 +4,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontrib
 
 go 1.22.0
 
-toolchain go1.22.6
+toolchain go1.22.4
```

Comment:
this is likely what's causing `make check` to fail in the CI, while not failing locally -- try manually reverting this